src/main/msp/msp.c

```diff
@@ -3646,7 +3646,7 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
         int32_t ned_vel_east = (int32_t)sbufReadU32(src);   // ned_vel_east
         gpsSol.groundSpeed = (uint16_t)sqrtf((ned_vel_north * ned_vel_north) + (ned_vel_east * ned_vel_east));
         (void)sbufReadU32(src);             // ned_vel_down
-        gpsSol.groundCourse = ((uint16_t)sbufReadU16(src) % 360);   // ground_course
+        gpsSol.groundCourse = ((uint16_t)sbufReadU16(src) % 36);   // ground_course
```

Comment:
I think they are sending 0-360 in whole degrees.
We use decidegrees (deg * 10) for `gpsSol.groundCourse`
To retain resolution, we should multiply by 10.  Maybe:
`gpsSol.groundCourse = ((uint16_t)sbufReadU16(src) % 360) * 10; // ground_course in decidegrees

Reply:
Done